src/main/java/com/minecolonies/coremod/entity/pathfinding/pathjobs/PathJobFindTree.java

```diff
@@ -154,7 +158,14 @@ private boolean isNearTree(@NotNull final Node n)
 
     private boolean isTree(final BlockPos pos)
     {
-        if (Tree.checkTree(world, pos, excludedTrees) && Tree.checkIfInColonyAndNotInBuilding(pos, colony))
+        final LivingEntity entity = this.entity.get();
+        BuildingLumberjack building = null;
```

Comment:
I ended up getting the lumberjack building from the entity. That seemed to be the easiest solution